### DISCUSSION TITLE
Next meeting

### DISCUSSION THREAD

**Discussion by baldimir** [dmn-tck/tck#508] [General]
Hi everyone, 

I will try to take stewardship of this project after Matteo. I see that our group didn't have a meeting for a long time, so it would be great to meet each other. I propose to do it e.g. on 15th of September, as now it is still holiday season. The time of the meeting could be the same as Matteo defined: 08:00am EDT(Toronto), 13:00 BST(London), 14:00 CEST (Milano), 22:00 AEDT (Melbourne). What do you think? Also let's use this thread for discussing proposals for the meeting, so we have everything at one place.

Thank you,
Tibor

**Comment by baldimir** [comment]
Proposal for the meeting agenda from me: 
- Stewardship agreement - I think we need to agree on a call about the stewardship. I wouldn't like to do it against anyone's will, so I would be glad if we can discuss it and agree for the future also on the call. 
- Retrospective - Discussion about how the group was working together, areas that worked well, that didn't work well. Basically to look back and improve. E.g. after discussion with Matteo, the timezones are a problem, so maybe we could think about working somehow without a meeting. 
- Highest priority issues and PRs. 

This is of course only my proposal, please comment if you like it or you would rather discuss something else.

**Comment by opatrascoiu** [comment]
Hi Tibor,
The date and time is good for me.
Thank you for organizing it.
Kind regards,Octavian
    On Tuesday, 9 August 2022 at 10:45:52 BST, Tibor Zimányi ***@***.***> wrote:  
 
 


Hi everyone,

I will try to take stewardship of this project after Matteo. I see that our group didn't have a meeting for a long time, so it would be great to meet each other. I propose to do it e.g. on 15th of September, as now it is still holiday season. The time of the meeting could be the same as Matteo defined: 08:00am EDT(Toronto), 13:00 BST(London), 14:00 CEST (Milano), 22:00 AEDT (Melbourne). What do you think? Also let's use this thread for discussion proposals for the meeting, so we have everything at one place.

Thank you,
Tibor

**Comment by StrayAlien** [comment]
Hi Tibor,

Thank you.  Apols for late reply - I've had some techy downtime.  That time
is good for me.

Greg.

**Comment by baldimir** [comment]
Thank you, invite sent, also to the mailing list.
